### ISSUE TITLE
never_kill_your_time should be installed on opt/ of linux 

### ISSUE THREAD

**Issue by rabingaire** [rabingaire/never_kill_your_time#16]

**Comment by nmunro** [comment]
Why not make it a snap package since it's designed to work on Ubuntu and wouldn't require you to have an install.sh or write anything to the root (/) file system?

**Comment by rabingaire** [comment]
I don't have much idea about snap package, But that's a good option I guess :)

**Comment by nmunro** [comment]
No-one has much idea about anything, until they start learning ;)

I don't know anything about snap either, for what it's worth, just throwing ideas at the wall and seeing what sticks for you.
